dependencies.yaml

```
@@ -4,7 +4,7 @@ files:
     output: conda
     matrix:
       cuda: ["11.8", "12.5"]
-      arch: [x86_64]
+      arch: [aarch64, x86_64]
```

I would prefer to revert this. We don't supply ARM environment files for most RAPIDS repos. If you'd like to add ARM conda envs for all of RAPIDS, I would do that in a separate PR and track the effort in rapidsai/build-planning for all repos.
```suggestion
      arch: [x86_64]
```

---

Given there are other users asking about ARM specifically with KvikIO (and running into issues without more explicit guidance), the environments provide a value add for them. For example: https://github.com/rapidsai/kvikio/issues/393

---

I suppose it's fine to include this. Maybe this should be done for all of RAPIDS, with a build-planning issue for tracking.

We don't have a good way to exclude (CUDA 11 + ARM), unless you create a separate file key entry for (CUDA 12 + ARM) That's not worth doing imo.

As we see increased ARM usage, it'll be somewhat helpful to have these environment files readily accessible. I have had to explain how to generate the ARM environments with `rapids-dependency-file-generator` before and it's a little complicated for new DFG users.

---

Agree this is a good idea

Have raised issue: https://github.com/rapidsai/build-planning/issues/87

Please add anything additional to that